Numsense/Chinese.fs

```diff
@@ -0,0 +1,194 @@
+﻿module internal Ploeh.Numsense.Chinese
+
+open Ploeh.Numsense.InternalDsl
+
+type CharacterSet =
+    {
+        negative: string;
+        zero: string;
+        one: string;
+        two: string;
+        alternativeTwo: string;
+        three: string;
+        four: string;
+        five: string;
+        six: string;
+        seven: string;
+        eight: string;
+        nine: string;
+        ten: string;
+        hundred: string;
+        thousand: string;
+        tenThousands: string;
+        hundredMillions: string
+    }
+
+let internal TraditionalCharacterSet =
```

Comment:
Since this is a value, it should be camelCased, i.e. `traditionalCharacterSet`.

Same reason as outlined above.